modules/button/react/lib/ButtonColors.ts

```diff
@@ -200,8 +200,10 @@ export const ButtonColors: ButtonColorCollection = {
     disabledBackground: 'transparent',
     disabledBorder: 'transparent',
     disabledText: 'rgba(8, 117, 225, 0.5)',
-    focusBackground: canvas.colors.blueberry100,
+    focusBackground: 'transparent',
     focusText: canvas.colors.blueberry400,
+    focusRingInner: 'currentColor',
```

Comment:
Since this will use the defaults, let's drop `focusRingInner` and `focusRingOuter` here.